cEP-0005.md

```diff
@@ -124,6 +124,17 @@ class RedundancyBear(LocalBear):
             yield Result.from_values(aspect=aspect, ...)
 ```
 
+Before moving any further, it is quite necessary to have a clear-cut definition of both aspects and tastes, to avoid any kind of discrepancies and chaos in the future.
+
+## aspects
+An aspect should just talk about a property of something, without qualifying it. It should be named something like `Length` rather than `TooLong`.
+Then it should be given the tastes. For e.g. aspect `Metadata.CommitMessage.Shortlog.Length` can have a taste like `max_shortlog_length`.
+
+## tastes
+tastes are the values that are meant to change the behaviour of their corresponding aspects. It should not be something like `allow` or `check` which turns the aspect value on or off, but it should be something more like a value which changes the behaviour of an aspect. It can be given as much context as needed but not more. For e.g. for the aspect `Metadata.CommitMessage.Shortlog`, set the taste `shortlog_colon` as true if it must have a colon in the shortlog, and false if the opposite is true.
```

Comment:
@adtac To make it more verbose and avoid taste naming clashes, so that you can for example request all suaspects of `Shortlog` by just defining to check `Metadata.CommitMessage.Shortlog`, or even the whole `Metadata.CommitMessage` subtree, and then just set a value for `shortlog_colon` without having to explicitly specify the `Shortlog.ColonExistence` aspect

**CC** @sils